src/components/application_manager/test/commands/mobile/speaker_request_test.cc

```diff
@@ -67,13 +67,21 @@ const uint32_t kKey = 5u;
 
 class SpeakRequestTest : public CommandRequestTest<CommandsTestMocks::kIsNice> {
  public:
-  SpeakRequestTest() : app_(CreateMockApp()) {
-    msg_ = CreateMessage(smart_objects::SmartType_Map);
+  SpeakRequestTest()
+      : app_(CreateMockApp())
+      , msg_(CreateMessage(smart_objects::SmartType_Map))
+      , mock_message_helper_(*am::MockMessageHelper::message_helper_mock()) {
+    testing::Mock::VerifyAndClearExpectations(&mock_message_helper_);
+  }
+
+  ~SpeakRequestTest() {
+    testing::Mock::VerifyAndClearExpectations(&mock_message_helper_);
```

Comment:
@LevchenkoS do we need it both n constructor and destructor? Please leave in one place